app/views/legislations/show.html.erb

```diff
@@ -11,170 +11,150 @@
                   </div>
-                  <%#= f.select(:content, [["Yes"],["No"]], {}, {:multiple => true}) %>
                   <% end %>
-                  </div>
-                </div>
-              <% end %>
 
-              <% unless clause.questions[3].answers.exists?(user_id: current_user.id) %>
-                <div data-target="slideshow.slide" class="slide">
-                  <%= clause.questions[3].content %>
-                  <div class="form-group">
-                    <%= form_for Answer.new, html: { id: "reportform" }, remote: true, update: { success: "response", failure: "error"} do |f| %>
-                    <%= f.hidden_field :user_id, :value => current_user.id  %>
-                    <%= f.hidden_field :question_id, :value => clause.questions[1].id  %>
-                    <%= f.text_area :content, :class => 'form-control', placeholder: clause.content %>
-                    <a data-action="slideshow#next">
-                      <%= f.submit 'suggest', :class => 'btn-primary' %>
-                    </a>
+                  <% unless clause.questions[1].answers.exists?(user_id: current_user.id) %>
+                    <div data-target="slideshow.slide" class="slide">
+                      <%= clause.questions[1].content %>
+                      <%= form_for @answer, html: { id: "reportform" }, remote: true, update: { success: "response", failure: "error"} do |f| %>
+                        <%= f.hidden_field :user_id, :value => current_user.id  %>
+                        <%= f.hidden_field :question_id, :value => clause.questions[1].id  %>
+                        <div class="buttonspacing">
+                          <%= f.select(:content, options_for_select([['1', '1'], ['2', '2'], ['3', '3'], ['4', '4'], ['5', '5']])) %>
+                          <a data-action="slideshow#next"><%= f.submit "submit", :class => 'btn-primary' %></a>
+                        </div>
+                      <% end %>
+                    </div>
                   <% end %>
-                  </div>
-                <% end %>
-
-                </div>
 
+                  <% unless clause.questions[2].answers.exists?(user_id: current_user.id) %>
+                    <div data-target="slideshow.slide" class="slide">
+                      <%= clause.questions[2].content %>
+                      <div class="form-group">
+                        <%= form_for Answer.new, html: { id: "reportform" }, remote: true, update: { success: "response", failure: "error"} do |f| %>
+                          <%= f.hidden_field :user_id, :value => current_user.id  %>
+                          <%= f.hidden_field :question_id, :value => clause.questions[1].id  %>
+                          <div class="buttonspacing">
+                           <%= f.select(:content, options_for_select([['No', 'No'], ['Yes', 'Yes']])) %>
+                           <a data-action="slideshow#next"><%= f.submit 'submit', :class => 'btn-primary' %></a>
```

Comment:
I have figured it out, it was :name => "content", which signifies params[:content] - so need to do that for all of them, and also made an edit to the answers controller params[:content] .... and then to fix the not being shown twice, I had to fix where the second two answers were storing, it was a copy/paste fuck up on my end, basically the answers were being stored as the same question_id as the second one, so that's why the first and second question wouldn't show when a user already answered them but the third and fourth would. The unless loop was working, but the answer was storing as the wrong thing -_-. Its all working and storing correctly now with the values from the buttons as the :content.